app/src/main/java/com/duckduckgo/app/systemsearch/SystemSearchActivity.kt

```diff
@@ -104,6 +111,9 @@ class SystemSearchActivity : DuckDuckGoActivity() {
     private lateinit var quickAccessAdapter: FavoritesQuickAccessAdapter
     private lateinit var itemTouchHelper: ItemTouchHelper
 
+    private var nestedScrollViewPosition: Int = 0
```

Comment:
Not to be addressed now, but do you think we could extract the autocomplete logic somehow so we don't need to duplicate it for BrowserTabFragment and SystemSearchActivity?

Reply:
I wanted to but since `SystemSearchActivity` uses a `NestedScrollView` and `BrowserTabFragment` uses a plain `RecyclerView`, I couldn’t easily share the logic.